src/crosscat_utils.py

```diff
@@ -462,16 +518,20 @@ def draw_state(bdb, table_name, generator_name, modelno,
 
             y += len(sorted_rows[view][cluster])
 
-        for i, row in enumerate(range(num_rows-1, -1, -1)):
+        assert len(row_idxs) == len(y_tick_labels)
+
+        for i, row in enumerate(range(maxrows-1, -1, -1)):
             if y_tick_labels[i] in hilight_rows:
                 fontcolor = row_hl_colors[y_tick_labels[i]]
                 fontsize = 'x-small'
                 fontweight = 'bold'
                 zorder = 10
             else:
                 fontsize = 'x-small'
-                fontcolor = '#333333'
-                fontweight = 'light'
+                rowidx = row_idxs[i]
+                highest_row = float(len(row_labels)) - 1
+                fontcolor = '#333333' # plt.cm.Set3(rowidx / highest_row)
```

Comment:
What was the commented-out code supposed to do? If it is not desired, remove it?